static/app/views/performance/transactionSummary/pageLayout.tsx

```diff
@@ -251,6 +251,12 @@ function PageLayout(props: Props) {
 
   const project = projects.find(p => p.id === projectId);
 
+  let hasWebVitals: ComponentProps<typeof TransactionHeader>['hasWebVitals'] =
```

Comment:
@JonasBa Yeah we can definitely do that, do you mean inefficient from a build time perspective?
Because in the end, typescript types aren't part of the final JS build right?